### PR TITLE
[MRG] FIX: reorient cell along z direction

### PR DESCRIPTION
**Pull request by jasmainak**
cc @rythorpe @cjayb 

Btw, should the basal dendrites be +ve z or -ve z direction?

### CODE REVIEW

File: hnn_core/viz.py [hunk 1]
```diff
@@ -553,16 +553,16 @@ def plot_cell_morphology(cell, ax, show=True):
...
         sec_pts = p_sec['sec_pts']
-        xs = [pt[0] for pt in sec_pts]
-        ys = [pt[1] for pt in sec_pts]
-        zs = [pt[2] for pt in sec_pts]
+        xs = [pt[0] + cell.pos[0] for pt in sec_pts]
+        ys = [pt[2] + cell.pos[1] for pt in sec_pts]
+        zs = [pt[1] + cell.pos[2] for pt in sec_pts]
         ax.plot(xs, ys, zs, 'b-', linewidth=linewidth)
-    ax.view_init(90, -90)
+    ax.view_init(0, -90)
```
Comment (jasmainak) [inline hnn_core/viz.py:565]: this verifies that [elevation](https://matplotlib.org/2.0.2/mpl_toolkits/mplot3d/api.html#mpl_toolkits.mplot3d.axes3d.Axes3D.view_init) is no longer required ... meaning `z` axis is along which the cell is oriented, not `y`

File: hnn_core/viz.py [hunk 2]
```diff
@@ -553,16 +553,23 @@ def plot_cell_morphology(cell, ax, show=True):
...
         linewidth = _linewidth_from_data_units(ax, p_sec['diam'])
         sec_pts = p_sec['sec_pts']
-        xs = [pt[0] for pt in sec_pts]
-        ys = [pt[1] for pt in sec_pts]
-        zs = [pt[2] for pt in sec_pts]
+        xs, ys, zs = list(), list(), list()
+        for pt in sec_pts:
+            dx = cell.pos[0] - cell.p_secs['soma']['sec_pts'][0][0]
+            dy = cell.pos[1] - cell.p_secs['soma']['sec_pts'][0][1]
+            dz = cell.pos[2] - cell.p_secs['soma']['sec_pts'][0][2]
```
Comment (jasmainak) [inline hnn_core/viz.py:567]: this is probably overkill but it helped me verify what I was doing is correct. You can potentially pass `ax` and plot all the cells in one plot this way.

Comment (rythorpe) [reply hnn_core/viz.py:567]: Good call.  This is a good sanity check.

**Review by rythorpe — APPROVED** [review]
Looks great @jasmainak! None of these parameters are stored in param files right?

**Comment by rythorpe** [comment]
Nevermind, I answered my own question. Merging now!

**Pull request merged**